### PR TITLE
Drop IE11 support and remove unmaintained depends

### PR DESCRIPTION
**Pull request by BryanQuigley**
Aside from letting us drop the legacy build it also let's us drop the jquery-deparam dependency replacing with built-in urlSearchParams

Confirmed this seems to work just fine - just not on IE11.

### CODE REVIEW

**Comment by BryanQuigley** [comment]
This is a prerequisite for further improvements (or removal) of jquery.   Confirmed through recent stats that IE is easily less than .1% (For reference Edge is at 6%)

File: _includes/footer.html [hunk 1]
```diff
@@ -1,5 +1,4 @@
 <!-- general behavior -->
-<script type="module" src="{{ site.baseurl }}/scripts/dist/modern/index.js"></script>
-<script nomodule src="{{ site.baseurl }}/scripts/dist/legacy/index.js"></script>
+<script type="module" src="{{ site.baseurl }}/scripts/dist/index.js"></script>
```
Comment (timwis) [inline _includes/footer.html:2]: The `package.json` still uses a named target. I would assume that `npm run build` still outputs the compiled `index.js` file to the `modern` directory, doesn't it? If not, feel free to disregard.

**Comment by timwis** [comment]
Hey Bryan, sorry for the delay. I'm happy with this change in principle - just left one question for you to confirm, then happy for you to merge when ready.

File: _includes/footer.html (hunk 1 above)
Comment (BryanQuigley) [reply _includes/footer.html:2]: Nope, it doesn't - and this was odd to me as well.

**Pull request merged**